### PR TITLE
CE procedures - PI to OCS data transfers

### PR DESCRIPTION
**Pull request by afloresOSI**
Output is available here: https://osisoft-dev.zoominsoftware.io/bundle/ocs-aflores-273956-pi-to-ocs-transfer-procedures/page/ocs-content-portal-overview.html
Vicki, there are a few inline comments that need to be addressed.

### CODE REVIEW

**Comment by VictoriaTouati-AVEVA** [comment]
> Output is available here: https://osisoft-dev.zoominsoftware.io/bundle/ocs-aflores-273956-pi-to-ocs-transfer-procedures/page/ocs-content-portal-overview.html Vicki, there are a few inline comments that need to be addressed.

@afloresOSI: I tried to address all inline comments via commit

**Review by VictoriaTouati-AVEVA — APPROVED** [review]

File: content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/transfer-data-to-ocs.md [hunk 1]
```diff
@@ -3,290 +3,265 @@ uid: transfer-data
...
 
    ![](../../images/transfer-settings.png)
    
-1. Enter a name and description for the transfer.
+1. In the **Name** and **Decription** fields, enter a name and description for the transfer.
 
-1. Optional: In the **Historical Start time** fields, enter a historical time context for the data retrieval.
+1. (Optional) In the **Historical Start time** fields, enter a historical time context for the data retrieval.
 
    **NOTE:** OCS supports out of order events and stores data in chronological order by timestamp. PI Data Archive 2017 SP2 or higher is required for this feature. Be sure to enter the historical start date and time correctly to ensure all data is included in the transfer. No data before the historical start time will be captured and stored in SDS.  See [Overview of data transfers](https://docs.osisoft.com/bundle/ocs/page/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/overview-data-transfers.html) to learn more about transferring historical data.
```
Comment (rmuolic) [inline content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/transfer-data-to-ocs.md:54]:
```suggestion
   **Note:** OCS supports out of order events and stores data in chronological order by timestamp. PI Data Archive 2017 SP2 or higher is required for this feature. Be sure to enter the historical start date and time correctly to ensure all data is included in the transfer. No data before the historical start time will be captured and stored in SDS.  See [Overview of data transfers](https://docs.osisoft.com/bundle/ocs/page/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/overview-data-transfers.html) to learn more about transferring historical data.
```

File: content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/transfer-data-to-ocs.md [hunk 2]
```diff
@@ -3,290 +3,265 @@ uid: transfer-data
...
+  1. Stop the transfer.
 
-2. Make modifications to the AF database and/or Data Archive digital state tables.
+  2. Make modifications to the AF database and/or Data Archive digital state tables.
 
-3. Restart the transfer to pick up these changes.
+  3. Restart the transfer to pick up these changes.
 
-### Start the data transfer
+## Start ae data transfer
```
Comment (rmuolic) [inline content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/transfer-data-to-ocs.md:247]: ae?

File: content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/confirm-data-retrieval.md [hunk 3]
```diff
@@ -4,20 +4,20 @@ uid: confirm-data-retrieval
...
+   **Note:** By default, the grid displays the SDS streams created by the PI to OCS data transfer. The grid lists the first 50 SDS streams in alphabetical order. You can change how many streams are displayed per page.
 
-1. Click the **Streams** down-arrow, then click **Types**.
+1. (Optional) In the **Search** field, enter search criteria to locate specific streams, and then press Enter.
 
-1. **Optional:** Enter search criteria to locate specific streams in the **Search** text box, then press Enter.
+1. (Optional) To view details about the stream metadata and type, select a stream.
 
-      **Result:** If there are matches, matching streams are shown.
+1. To review the types created, in the **Streams** dropdown, select **Types**.
```
Comment (rmuolic) [inline content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/confirm-data-retrieval.md:21]:
```suggestion
1. To review the types created, in the **Streams** dropdown list, select **Types**.
```

File: content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/transfer-data-to-ocs.md (hunk 2 above)
Comment (rmuolic) [inline content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/transfer-data-to-ocs.md:247]:
```suggestion
## Start a data transfer
```

File: content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/transfer-data-to-ocs.md [hunk 4]
```diff
@@ -3,290 +3,265 @@ uid: transfer-data
...
-1. Click **Ok**.
+1. Select **Ok**.
 
-   **Result:** The transfer is created and the `Transfer` page opens.
-
-   ![](../../images/transfer-win.png)
+   The transfer is created and the `Transfer` page opens.
 
-**Note:** You can click the **Settings** button to access the **Transfer Setting** dialog box and modify transfer settings (name, description, historical start and end times, and data privacy settings).
+**Note:** To modify the transfer settings, select **Settings** to access the `Transfer Setting` window and modify thename, description, historical start and end times, and data privacy settings as needed.
```
Comment (rmuolic) [inline content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/transfer-data-to-ocs.md:67]:
```suggestion
**Note:** To modify the transfer settings, select **Settings** to access the `Transfer Setting` window and modify the name, description, historical start and end times, and data privacy settings as needed.
```

File: content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/transfer-data-to-ocs.md [hunk 5]
```diff
@@ -3,290 +3,265 @@ uid: transfer-data
...
+1. On the `Transfer` page, select the **AF Database** down arrow, then select the source AF database to search.
 
-1. Optional: In the `Select Root Element` dialog box, click the plus buttons to drill down to the desired root asset in the AF database hierarchy, select a root asset, then click **Select**. 
+   ![](../../images/transfer-win.png)
 
-1. Optional: In the **Element Name** text box, enter search criteria to filter by part or all of an AF element name.
+1. (Optional) Select **Root Asset**.
 
-   **Note:** If you don't enter filter criteria, the search defaults to "*" or all.  
+1. (Optional) In the `Select Root Element` window, select the plus buttons to drill down to the root asset in the AF database hierarchy, select a root asset, then select **Select**. <!-- AF 11/15/21 UGH! I hate "select **Select**". Makes me want to use click. -->
```
Comment (rmuolic) [inline content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/transfer-data-to-ocs.md:81]: If it helps, you can use "choose".

```suggestion
1. (Optional) In the `Select Root Element` window, select the plus buttons to drill down to the root asset in the AF database hierarchy, select a root asset, then choose **Select**. <!-- AF 11/15/21 UGH! I hate "select **Select**". Makes me want to use click. -->
```

File: content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/transfer-data-to-ocs.md [hunk 6]
```diff
@@ -3,290 +3,265 @@ uid: transfer-data
...
-1. In the `Details` pane, click **Create Transfer**.
+1. In the `Details` pane, select **Create Transfer**.
 
-   **Result:** The **Transfer Setting** dialog box opens.
+   The `Transfer Setting` window opens.
 
    ![](../../images/transfer-settings.png)
    
-1. Enter a name and description for the transfer.
+1. In the **Name** and **Decription** fields, enter a name and description for the transfer.
```
Comment (rmuolic) [inline content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/transfer-data-to-ocs.md:50]:
```suggestion
1. In the **Name** and **Description** fields, enter a name and description for the transfer.
```

**Review by rmuolic — CHANGES_REQUESTED** [review]
Minor suggested corrections.

**Pull request merged**